samples/Web/deploy-webapp-ip-restrictions/azurepolicy.json

```diff
@@ -0,0 +1,75 @@
+{
+    "properties": {
+        "displayName": "Web App default IP restrictions",
+        "mode": "indexed",
+        "description": "Configures Web App IP restriction to allow access from specified public IP addresses",
+        "metadata": {
+            "category": "Web"
+        },
+        "parameters": {},
+        "policyRule": {
+            "if": {
+                "field": "type",
+                "equals": "Microsoft.Web/sites"
+            },
+            "then": {
+                "effect": "DeployIfNotExists",
+                "details": {
+                    "type": "Microsoft.Web/sites/config",
+                    "roleDefinitionIds": [
+                        "/providers/Microsoft.Authorization/roleDefinitions/de139f84-1756-47ae-9be6-808fbbe84772"
+                    ],
+                    "existenceCondition": {
+                        "not": {
+                            "field": "Microsoft.Web/sites/config/web.ipSecurityRestrictions[*].action",
+                            "Equals": "Allow"
+                        }
+                    },
+                    "deployment": {
+                        "properties": {
+                            "mode": "incremental",
+                            "parameters": {
+                                "name": {
+                                    "value": "[field('name')]"
+                                }
+                            },
+                            "template": {
+                                "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                                "contentVersion": "1.0.0.0",
+                                "parameters": {
+                                    "name": {
+                                        "type": "string"
+                                    }
+                                },
+                                "resources": [
+                                    {
+                                        "name": "[concat(parameters('name'), '/web')]",
+                                        "type": "Microsoft.Web/sites/config",
+                                        "apiVersion": "2018-11-01",
+                                        "properties": {
```

Comment:
>"properties": { [](start = 40, length = 15)

what happens to the rest of the properties in config/web?  Do they revert to their default values or keep the values they had prior to this deployment?